### ISSUE TITLE
Shifted bounding boxes after inference

### ISSUE THREAD

**Issue by 9friday** [stephansturges/WALDO#14]
Hi, 
I am trying to run inference using a custom trained YOLOv7 model in both the .pt and .onnx formats. 

The bounding boxes and confidence values from the weight files of both the formats aren't the same. It appears that the bounding boxes are shifted. 
Both the pytorch inference code and the onnx inference code is written using Python.

I am attaching pictures from both pytorch and onnx inference on one of the images below. 
Pytorch inference : 
![objectInWild_AerialVid12-fNo12401](https://github.com/stephansturges/WALDO/assets/37792181/b1853c06-adbd-4d1d-9256-f100faeb7e87)
Onnx inference : 
![objectInWild_AerialVid12-fNo12401](https://github.com/stephansturges/WALDO/assets/37792181/271006fb-1aa5-4660-9627-98d1e3cafafa)

To recreate the issue :
Pytorch inference : 
I used the detect.py file from the [yolov7](https://github.com/WongKinYiu/yolov7.git) repository. 
Convert the model from pytorch to onnx: 
I used the export.py file from the yolov7<link to repo> repository with the following command line arguments :  
python3 export.py --weights /custom model_path/yolov7_custom_best.pt --grid --end2end --simplify --topk-all 100 --iou-thres 0.50 --conf-thres 0.2 --img-size 960 540 --max-wh 960. 
The commit id : 84932d70fb9e2932d0a70e4a1f02a1d6dd1dd6ca

ONNX Inference :
I used the [WALDO](https://github.com/stephansturges/WALDO.git) repository. 
The commit id : a99ad40be13395d3d7cf67bf3704e122862658ca
The playground/run_local_network_on_images_onnxruntime.py file was only modified to log inference times. 

I am unable to identify what could be causing this issue and would appreciate any insight regarding why this issue could occur and how to rectify it. Thanks in advance!

**Comment by stephansturges** [comment]
yes you're right there is an offset that is not corrected well there.. I will fix it in the next release
